config/initializers/preview_generation.rb

```diff
@@ -0,0 +1,11 @@
+ActiveStorage::Previewer::VideoPreviewer.class_eval do
```

Comment:
I'd prefer we extended this class using a file in `lib/` - e.g. `lib/active_storage/previewer/video_previewer.rb`, and required it in an initializer just for activestorage - `config/initializers/active_storage.rb`. 

Extending a class in this way can have side-effects since initialisation can happen or not happen at different times, and it also means that there's a single implementation for this file. Generally my understanding of `lib/` is that it is _for_ this kind of code - extensions or non-Rails code.

You should also not need `class_eval` - just reopenening the class should be sufficient.

Let me know if you need any help with any of the above.